### ISSUE TITLE
[FEATURE] Common denominator (unified) set of tenses.

### ISSUE THREAD

**Issue by ncoder** [RosaeNLG/rosaenlg#73]
**Is your feature request related to a problem? Please describe.**

It would be nice to have a "least common denominator" set of tenses, for multi-lingual use-cases.

There is currently no intersection between Spanish tenses and English tenses:

```js
export type Tense =
    // english
    'SIMPLE_PAST' | 'PAST' |
    'SIMPLE_PRESENT' | 'PRESENT' |
    'SIMPLE_FUTURE' | 'FUTURE' |
    'PROGRESSIVE_PAST' |
    'PROGRESSIVE_PRESENT' |
    'PROGRESSIVE_FUTURE' |
    'PERFECT_PAST' |
    'PERFECT_PRESENT' |
    'PERFECT_FUTURE' |
    'PERFECT_PROGRESSIVE_PAST' |
    'PERFECT_PROGRESSIVE_PRESENT' |
    'PERFECT_PROGRESSIVE_FUTURE' |
    // spanish
    'INDICATIVE_PRESENT' |
    'INDICATIVE_IMPERFECT' |
    'INDICATIVE_PRETERITE' |
    'INDICATIVE_PERFECT' |
    'INDICATIVE_PLUPERFECT' |
    'INDICATIVE_FUTURE_PERFECT' |
    'INDICATIVE_PRETERITE_PERFECT' |
    'SUBJUNCTIVE_PRESENT' |
    'SUBJUNCTIVE_IMPERFECT_RA' |
    'SUBJUNCTIVE_IMPERFECT_SE' |
    'SUBJUNCTIVE_FUTURE' |
    'SUBJUNCTIVE_PERFECT' |
    'SUBJUNCTIVE_PLUPERFECT' |
    'SUBJUNCTIVE_FUTURE_PERFECT' |
    'CONDITIONAL_PRESENT' |
    'CONDITIONAL_PERFECT';
```
I'm assuming a similar problem exists for other languages.

**Describe the solution you'd like**
A solution proposed is to have 
UNIVERSAL_xxx enums, which each language wrapper would be able to map.
I'm a bit more partial to re-using some of the existing tenses, if possible, like english has simply PAST | PRESENT | FUTURE.
But i think a linguist should weigh in here to tell us what makes sense and won't cause confusion.

**Additional context**
#72

**Comment by ludans** [comment]
I'm not a linguist, but let's draft something.

Would that make sense at least for English and Spanish?

[tenses.pdf](https://github.com/RosaeNLG/rosaenlg/files/6412144/tenses.pdf)

**Comment by ludans** [comment]
implemented in https://github.com/RosaeNLG/rosaenlg/tree/utenses

**Comment by ncoder** [comment]
Very nice! I can't comment on it's accuracy; But that's pretty awesome.

**Comment by ludans** [comment]
is available in version 3.0.0
